readme.md

```
@@ -20,7 +20,7 @@ The official client for [Forged Alliance Forever (FAF)](https://www.faforever.co
 1. Select `Main` as run configuration next to the hammer button in the top right
 1. Compile and start the application by pressing the play button
 
-A video tutorial is available [here](https://youtu.be/6gsHnt02I_Y). Don't forget to `Enable annotation processing`.
+A video tutorial is available [here](https://www.youtube.com/watch?v=4Wm-yZpV_q8). Don't forget to `Enable annotation processing`.
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)